packages/gdk_pixbuf.rb

```diff
@@ -3,67 +3,63 @@
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gdk_pixbuf-2.40.0-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gdk_pixbuf-2.40.0-1-chromeos-armv7l.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gdk_pixbuf-2.40.0-1-chromeos-x86_64.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gdk_pixbuf-2.40.0-1-chromeos-i686.tar.xz',
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gdk_pixbuf-2.42.2-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gdk_pixbuf-2.42.2-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/gdk_pixbuf-2.42.2-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gdk_pixbuf-2.42.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '1ee4c25a59579f11c12b3ef0f2b466710268164e4810c879861aa65a4185edfb',
-     armv7l: '1ee4c25a59579f11c12b3ef0f2b466710268164e4810c879861aa65a4185edfb',
-     x86_64: 'b8ab8af77ef2b41003487ea06f17ffa8e3828af62b646aa634b8a092160df1ea',
-       i686: 'e041085afe17c2a853ef5057e44c05e3e3b0e4b943557f1d60e59b574ee52bf3',
+     aarch64: '3db50fb39c83c5b61f4e27efbfedeaa0ea2706f7ef1f059e32d0f06ab712c93f',
+      armv7l: '3db50fb39c83c5b61f4e27efbfedeaa0ea2706f7ef1f059e32d0f06ab712c93f',
+        i686: '487395d7623101d17960ba51739e535a82f7362388c49bacc87449a56d7c3fe2',
+      x86_64: '39f8ce2eb21d95bad9c5dbc68424374d94013d60681e03d4943d86d27c713e02',
   })
 
-  depends_on 'meson' => :build
   depends_on 'pango'
   depends_on 'gobject_introspection'
   depends_on 'jasper'
   depends_on 'libtiff'
   depends_on 'libjpeg_turbo'
+  depends_on 'libwebp'
   depends_on 'six'
 
   def self.build
-    Dir.mkdir 'build'
-    Dir.chdir 'build' do
-      system 'meson',
-             "--prefix=#{CREW_PREFIX}",
-             "--libdir=#{CREW_LIB_PREFIX}",
-             "--mandir=#{CREW_PREFIX}/share/man",
-             '-Dinstalled_tests=false',
-             '-Dbuildtype=release',
-             '-Drelocatable=true',
-             '-Djasper=true',
-             '-Ddebug=false',
-             '-Dman=false'
-      system 'ninja'
-    end
+    system "meson #{CREW_MESON_LTO_OPTIONS} \
+      -Dinstalled_tests=false \
+      -Dbuiltin_loaders=all \
+      -Drelocatable=true \
+      -Djasper=true \
+      -Ddebug=false \
+      -Dman=false \
+      builddir"
+    system "meson configure builddir"
+    system "ninja -C builddir"
   end
 
   def self.install
-    Dir.chdir 'build' do
-      system "DESTDIR=#{CREW_DEST_DIR} ninja install"
-    end
+    system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
     loader_dir = "#{CREW_DEST_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders"
-    FileUtils.mkdir_p "#{loader_dir}"
+    FileUtils.mkdir_p loader_dir
     system "touch #{loader_dir}.cache"
   end
 
   def self.postinstall
     ENV['GDK_PIXBUF_MODULEDIR'] = "#{CREW_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders"
```

Comment:
Can we remove this? (Since we will do it in `puts "source ~/.bashrc".orange`)

Reply:
These `ENV` variables are needed for:
```
 system 'gdk-pixbuf-query-loaders --update-cache'
```